package.json

```diff
@@ -5,6 +5,7 @@
     "babel-preset-es2015": "^6.3.13",
     "bluebird": "^3.0.5",
     "body-parser": "^1.14.1",
+    "co": "^4.6.0",
```

Comment:
I'm using co because it is tiny and bluebird was breaking Promise, (was getting weird error Promise.all not defined!)